docs/metrics/pr-curve.md

```diff
@@ -151,14 +153,12 @@ can visually conclude which curves indicate that a certain class or model has a
 better performance.
 
 <center>
-![pr.png](../assets/images/metrics-prcurve.png)
+![pr.png](../assets/images/metrics-prcurve-light.png#only-light)
+![pr.png](../assets/images/metrics-prcurve-dark.png#only-dark)
 </center>
 
-In the plot above, we see that the blue curve has a higher precision than the orange curve for almost every recall
-value. This means that the model behind the blue curve performs better. There are many factors that might make the
-blue curve more jagged compared to the orange curve, such as having a lower number of thresholds (fewer points), poorly
```

Comment:
Curious about why the additional discussion gets dropped.

Reply:
Thanks for flagging this.  I wanted to double-check with @marko-polo-cheno on this change. 
@marko-polo-cheno, do you see a problem with dropping this section?  Mainly curious to see if this was added because the plot was jagged or you intentionally plotted a jagged PR curve to point this out.

Reply:
For reference this was the original plot I made to focus a bit more on the fact that PR curves are not guaranteed to be monotonically decreasing. But the new plots look better and so the old text is obsolete
<img width="577" alt="image" src="https://github.com/kolenaIO/kolena/assets/37851975/702df148-b2c6-4d0b-be5a-1df984e07e17">

👍